src/org/labkey/test/components/ui/entities/EntityInsertPanel.java

```diff
@@ -52,7 +52,16 @@ public WebDriver getDriver()
 
     public ReactSelect targetEntityTypeSelect()
     {
-        return ReactSelect.finder(getDriver()).withName("targetEntityType").waitFor();
+        // Which tabs are available and selected can vary so try finding the visible react select
+        ReactSelect firstTabSelect = ReactSelect.finder(getDriver()).withName("targetEntityType").index(0).waitFor();
```

Comment:
rather than finding by index and testing for .isDisplayed(), perhaps .notHidden() will work?
Unfortunately, searching by index without limiting scope can return you a reactSelect that might not be within the bounds of the component.  Definitely consider supplying the searchContext in the .waitFor(getComponentElement()) call

Reply:
I wasn't able to get that to work. Couldn't get this to work without checking the first two indexes (one for the two possible tabs)